scripts/metric_reporter/reporter/suite_reporter.py

```diff
@@ -285,7 +285,7 @@ def _parse_artifacts(
                 # Update date and timestamp at the test_suites level if not already set
                 if not test_suite_result.date and suites.timestamp:
                     test_suite_result.timestamp = suites.timestamp
-                    test_suite_result.date = suites.timestamp.split("T")[0]
+                    test_suite_result.date = self._extract_date(suites.timestamp)
```

Comment:
Sorry, trying to follow the threads here... before we split on `T`; which means neutering the times and just keeping a UTC date.
Is the assumption that we'd never log this more than once per day, and therefore don't need times? Or what happens if this runs multiple times per day and we might not have times to know which is the first/middle/last run for the day?

Reply:
Essentially this value is created just to make nicer graphs in google sheets.

All the models have both date and timestamp, but timestamp is too verbose for graphing in the sheets, so I created a date column. This could mean multiple columns with the same date, but most of the time there is too much data for that to happen. The alternative was job number, but this was less informative.